docs/docs/how-to-deploy.mdx

```diff
@@ -94,16 +94,8 @@ that are part of your action code but not included in the base `rasa/rasa-sdk` i
 
 To create your image:
 
-1. Move your actions code to a folder `actions` in your project directory.
-   Make sure to also add an empty `actions/__init__.py` file:
-
-   ```bash
-   mkdir actions
-   mv actions.py actions/actions.py
-   touch actions/__init__.py  # the init file indicates actions.py is a python module
-   ```
-
-   The `rasa/rasa-sdk` image will automatically look for the actions in `actions/actions.py`.
+1. Make sure your actions are defined in `actions/actions.py`. The `rasa/rasa-sdk`
+  image will automatically look for the actions in this file.
```

Comment:
Just a thought, can you provide multiple actions files? If so it probably makes sense to update rasa sdk to pass the directory instead of that specific filename

Reply:
but we're using `DEFAULT_ACTIONS_PATH` which is just `actions`, so Rasa SDK will by default load the whole `actions` package

Reply:
sorry, should have clarified, i meant in the dockerfile: https://github.com/RasaHQ/rasa-sdk/blob/master/Dockerfile#L40

Reply:
aah got it, good point! we should change that